otter/rest/groups.py

```diff
@@ -6,15 +6,22 @@
 
 from functools import partial
 
+from twisted.internet import reactor
```

Comment:
I think both `CassScalingGroup` and `CassScalingGroupCollection` have a reactor on the objects, which can be passed to `get_active_cache` rather than importing the global reactor here.